pytest/tests/sanity/epoch_sync.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
```

Comment:
This is fine but you can consider using unitttest like so:
https://github.com/near/nearcore/blob/master/pytest/README.md#creating-new-tests